setup.py

```diff
@@ -77,8 +77,8 @@
     
     dicompyler requires the following packages to run from source:
     
-    - Python 2.7 or higher (not tested on Python 3)
-    - wxPython 3.0.2.0 (not tested on wxPython Phoenix)
+    - Python 3.6 or higher 
```

Comment:
Will Python 3.5 work as well? I just upgrade my system to Python 3.6, but some other workstations are using Python 3.5.